Add a simple launch file to load robot description

Tiryoh opened the pull request:
<!-- プルリクエストのタイトルと以下の記述項目は、日本語で書いても良いです -->

# What does this implement/fix?
<!-- Explain your changes here. -->
<!-- このPRはどんな機能改善/修正ですか？ -->

リンク情報を取得したい場合にRVizの起動は不要なのでxacroのみをロードするlaunchを作成します。


# Does this close any currently open issues?
<!-- このPRはオープンになっているissueをクローズしますか？ -->

しません。

# How has this been tested?
<!-- このPRはどのようにテストしましたか？ -->

`roslaunch raspimouse_description display_xacro.launch`が起動することを確認しています

`roslaunch raspimouse_description display_xacro.launch use_rviz:=false`を実行後にRVizを手動で起動してRobotModelが表示されることを確認しています

# Any other comments?
<!-- その他コメントはありますか？ -->

なし

# Checklists
<!-- PR作成時にチェックボックスにチェックを入れてください -->

- [x] <!-- コントリビューティングガイドラインを読みました--> I have read the CONTRIBUTING guidelines.
- [x] <!-- 同じ変更を要求するオープンなPRが無いことを確認しました --> I have checked to ensure there aren't other open [Pull Requests](../pulls) for the same change.

Tiryoh committed:
Add a simple launch file to load robot description

ShotaAk commented:
@Tiryoh 新しくlaunchファイルを作成せず、今あるlaunchファイルに引数を追加して、
`<group if="$(arg use_rviz)"></group>` でrvizの起動をON/OFFしたいのですが、どうでしょうか？

Tiryoh committed:
Merge into single launch file

Tiryoh committed:
Refactor

lowercase letters are introduced as an example
https://wiki.ros.org/roslaunch/XML#if_and_unless_attributes

Tiryoh commented:
* 統合するメリット
    * 階層が深くならないので確認しやすい
* 統合するデメリット
    * ファイル名やオプションの名前だけでは使い方がわかりにくい

デメリットに関してはREADMEへの追加やサンプルの充実で解決できると思うので統合します

ShotaAk reviewed (approved):
LGTM
